src/styles/pages/StoreItemPage.module.scss

```diff
@@ -0,0 +1,44 @@
+.optionsContainer {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+.rowContainer {
+  align-items: flex-start;
+  align-self: center;
+  display: flex;
+  flex-direction: row;
+  gap: 5rem;
+  justify-content: center;
+  width: 95%;
+
+  @media (max-width: 600px) {
```

Comment:
I think the image can get pretty small before converting into a single column layout. I would increase this `max-width` so the single column layout kicks in earlier

![image](https://github.com/acmucsd/membership-portal-ui-v2/assets/22133785/e2a23669-0194-42c2-9d85-d176711b562c)

This codebase would prefer you use the media breakpoint variables in vars.scss for consistency,  rather than hard-coded values like `600px`. You can search for `@media` in other files to see how they're used